scala/scala.bzl

```diff
@@ -260,60 +253,57 @@ scala_repl = rule(
   )
-  native.http_jar(
-      name = "scalactic_2_11",
-      url =
-      "http://central.maven.org/maven2/org/scalactic/scalactic_2.11/3.0.5/scalactic_2.11-3.0.5.jar",
-      sha256 = "84723064f5716f38990fe6e65468aa39700c725484efceef015771d267341cf2"
+  _scala_maven_import_external(
+      name = "io_bazel_rules_scala_scalactic",
+      artifact = "org.scalactic:scalactci_{major_version}:3.0.5".format(
+          major_version = major_version),
+      jar_sha256 = scala_jar_shas[major_version]["scalactic"],
+      licenses = ["notice"],
+      server_urls = maven_servers,
   )
 
-  native.http_jar(
-      name = "scalatest_2_12",
-      url =
-      "http://central.maven.org/maven2/org/scalatest/scalatest_2.12/3.0.5/scalatest_2.12-3.0.5.jar",
-      sha256 = "b416b5bcef6720da469a8d8a5726e457fc2d1cd5d316e1bc283aa75a2ae005e5"
+  _scala_maven_import_external(
+      name = "io_bazel_rules_scala_scala_xml",
+      artifact = "org.scala-lang.modules:scala-xml_{major_version}:1.0.5".format(major_version = major_version),
+      jar_sha256 = scala_jar_shas[major_version]["scala_xml"],
+      licenses = ["notice"],
+      server_urls = maven_servers,
   )
-  native.http_jar(
-      name = "scalactic_2_12",
-      url =
-      "http://central.maven.org/maven2/org/scalactic/scalactic_2.12/3.0.5/scalactic_2.12-3.0.5.jar",
-      sha256 = "57e25b4fd969b1758fe042595112c874dfea99dca5cc48eebe07ac38772a0c41"
-  )
-  native.http_jar(
-      name = "scala_xml_2_11",
-      url =
-      "http://central.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.5/scala-xml_2.11-1.0.5.jar",
-      sha256 = "767e11f33eddcd506980f0ff213f9d553a6a21802e3be1330345f62f7ee3d50f"
-  )
-  native.http_jar(
-      name = "scala_xml_2_12",
-      url =
-      "http://central.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.5/scala-xml_2.12-1.0.5.jar",
-      sha256 = "035015366f54f403d076d95f4529ce9eeaf544064dbc17c2d10e4f5908ef4256"
-  )
-  native.http_jar(
-      name = "scala_parser_combinators_2_11",
-      url =
-      "http://central.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.11/1.0.4/scala-parser-combinators_2.11-1.0.4.jar",
-      sha256 = "0dfaafce29a9a245b0a9180ec2c1073d2bd8f0330f03a9f1f6a74d1bc83f62d6"
-  )
-  native.http_jar(
-      name = "scala_parser_combinators_2_12",
-      url =
-      "http://central.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.12/1.0.4/scala-parser-combinators_2.12-1.0.4.jar",
-      sha256 = "282c78d064d3e8f09b3663190d9494b85e0bb7d96b0da05994fe994384d96111"
+
+  _scala_maven_import_external(
+      name = "io_bazel_rules_scala_scala_parser_combinators",
+      artifact = "org.scala-lang.modules:scala-parser-combinators_{major_version}:1.0.4".format(major_version = major_version),
+      jar_sha256 = scala_jar_shas[major_version]["scala_parser_combinators"],
+      licenses = ["notice"],
+      server_urls = maven_servers,
   )
 
   native.maven_server(
```

Comment:
if we don't use maven_jar then we don't need maven_server anymore

Reply:
That `maven_server` is still used by 
```
  native.maven_jar(
      name = "scalac_rules_protobuf_java",
      artifact = "com.google.protobuf:protobuf-java:3.1.0",
      sha1 = "e13484d9da178399d32d2d27ee21a77cfb4b7873",
      server = "scalac_deps_maven_server",
  )
```

Could remove it if we're happy just using maven central instead of the bazel mirror

Reply:
Let’s keep it. Maybe we should drop the maven jar entirely but that’s a different story

Reply:
Let’s keep it. Maybe we should drop the maven jar entirely but that’s a different story